android-components/components/feature/fxsuggest/src/test/java/mozilla/components/feature/fxsuggest/FxSuggestFactsTest.kt

```diff
@@ -15,17 +15,18 @@ import org.junit.Test
 class FxSuggestFactsTest {
 
     @Test
-    fun testEmitSponsoredSuggestionClickedFact() {
+    fun emitSuggestionClickedFactWithAmpInteractionInfo() {
```

Comment:
Just for future reference, we are trying to name all our tests with the format of `GIVEN <x> WHEN <y> THEN <z>` within backticks. We're doing this even when adding tests to old files with different naming conventions. Since there isn't really consistency throughout the code base yet I don't feel strongly about whether you actually update these though.

Also worth noting that last I checked tests in the `src/androidTest` directories cannot use the Kotlin backtick function naming though so there's at least one exception 😅

Reply:
Thanks for the heads-up about `androidTest`, and the new naming convention! I'll go ahead and change them now, since this is all new code in this component.